Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "project_ares"
-version = "0.9.0"
+version = "0.10.0"
```

Comment:
I thought the next release would be `1.0`

Reply:
why

Reply:
Nevermind, `1.0` isn't the next release.